internal/service/cloudbackupschedule/resource_cloud_backup_schedule.go

```diff
@@ -492,9 +492,9 @@ func cloudBackupScheduleCreateOrUpdate(ctx context.Context, connV2 *admin.APICli
 	}
 
 	req := &admin.DiskBackupSnapshotSchedule{}
-
-	if v, ok := d.GetOk("copy_settings"); ok && len(v.([]any)) > 0 {
-		req.CopySettings = expandCopySettings(v.([]any))
+	copySettings := d.Get("copy_settings")
```

Comment:
The only time `d.Get("copy_settings")` will return `nil` is when `copy_settings` doesn't exist in the schema.